pfcpiface/node.go

```diff
@@ -55,6 +55,23 @@ func (node *PFCPNode) handleNewPeers() {
 	lAddrStr := node.LocalAddr().String()
 	log.Infoln("listening for new PFCP connections on", lAddrStr)
 
+	for _, peer := range node.upf.peers {
+		n4DstIP, err := getRemoteIP(peer)
```

Comment:
the body of `for` is too long IMO. I propose to create a new function e.g. called `tryConnectToN4Peer()`.

Reply:
Done